src/utils/apis/OfferApi.js

```diff
@@ -88,15 +84,41 @@ export const offersAPI = {
 
     },
     getOffersFromHr() {
-        return this.getAllOffers(); //TODO change after full users functionality implementation
+        return fetch(recruitmentServiceBasicAPILink + `/api/hrpartners/offers`, {
```

Comment:
To działa? Z tego, co widzę, to wszędzie jest zapis `<function_name>: function(){}`, a tu masz tak po prostu?

Reply:
działa